test/specs/lib/numberToWord-test.js

```diff
@@ -26,7 +26,7 @@ describe('numberToWord', () => {
   })
 
   it('returns word if input is word', () => {
-    words.forEach((word) => numberToWord(word).should.equal(word))
+    words.forEach(word => numberToWord(word).should.equal(word))
```

Comment:
Why is it that in some (most?) cases the parens are added, but in others they are removed?

https://github.com/Semantic-Org/Semantic-UI-React/pull/1939/files#diff-ca99bb99c3b7ee63716bc6db755e5413R107

https://github.com/Semantic-Org/Semantic-UI-React/pull/1939/files#diff-ca99bb99c3b7ee63716bc6db755e5413R107

..etc